app/Http/Controllers/Api/WaitingRoomsController.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Gladiator\Http\Controllers\Api;
+
+use Gladiator\Models\WaitingRoom;
+use Illuminate\Http\Request;
+
+class WaitingRoomsController extends ApiController
+{
+    public function calculateSplit(Request $request)
```

Comment:
So if we're using the internal controller, do we still care at all about API keys?

If we do care, one option to is embed both the API URL in a data attribute as @weerd suggested but ALSO embed an api key. You'd only be able to read that if you were able to access the page.

But again not really sure what the risk is here. Northstar user id's are exposed everywhere publicly, and you can't actually split this way, it just returns hypothetical match ups.